operator/roles/kiali-deploy/defaults/main.yml

```diff
@@ -23,6 +23,7 @@ kiali_defaults:
 
   auth:
     strategy: ""
+    oauth_name: ""
```

Comment:
trivial - All of these settings are alphabetically ordered. Keep that convention - move `oauth_name` above `strategy`.